cpp/src/metrics/metrics.cu

```diff
@@ -38,5 +39,9 @@ namespace ML {
             return MLCommon::Metrics::computeRandIndex(y, y_hat, (uint64_t)n, handle.getDeviceAllocator(), handle.getStream());
```

Comment:
Ouch.. my bad.. I should have noticed this one in your last PR!

`y` and `y_hat` are supposed to be of some integer types, right? Why are they double here?